tasks/routes/users.js

```diff
@@ -0,0 +1,89 @@
+const express = require('express');
+const uuid = require('uuid');
+
+const router = express.Router();
+
+const userStorage = new Map();
+
+function User(login, password, age, uid = uuid.v4()) {
+    this.id = uid;
+    this.login = login;
+    this.password = password;
+    this.age = age;
+    this.isDeleted = false;
+}
+
+const defaultUser = new User('default', 'password', 25, '1');
+const secondUser = new User('secondUser', 'password', 14, '2');
+const thirdUser = new User('thirdUser', 'password', 38, '3');
+userStorage.set(defaultUser.id, defaultUser);
+userStorage.set(secondUser.id, secondUser);
+userStorage.set(thirdUser.id, thirdUser);
+
+function isUserExist(request) {
+    return userStorage.has(request.params.id);
+}
+
+router.get('/user/:id', (request, response) => {
```

Comment:
Optionally, you could also create a separate "controllers" folder and keep the callback logic of each route in there under the respectively named files.
This would help to further improve the level of abstraction.